packages/experimental/Switch/src/__tests__/__snapshots__/Switch.test.tsx.snap

```diff
@@ -168,7 +168,7 @@ exports[`Switch Disabled 1`] = `
         ],
       },
       "trackBackgroundStyle": Object {
-        "backgroundColor": "rgba(180, 214, 250, 1)",
+        "backgroundColor": "rgba(240, 240, 240, 1)",
```

Comment:
This results in the change as before it was taking initial animation value before moving to the correct state , Now it take correct animation value directly.